conda_smithy/lint_recipe.py

```diff
@@ -60,6 +63,20 @@ def get_section(parent, name, lints):
     return section
 
 
+def get_source_section(parent, lints):
+    section = parent.get('source', {})
+    if isinstance(section, dict):
+        return [ section ]
+    elif (isinstance(section, ruamel.yaml.comments.CommentedSeq) or
+          isinstance(section, list)):
```

Comment:
When parsing the `meta.yaml` the `section` is always a `CommentedSeq`, but during tests, it is easier to pass a `list`.

Reply:
Isn't CommentedSeq inherited from list?